libmscore/stringdata.cpp

```diff
@@ -67,6 +75,8 @@ void StringData::read(XmlReader& e)
             else
                   e.unknown();
             }
+      if (stringTable.size() == 5) 
```

Comment:
What about other plucked string instruments with 5 strings, like some basses?

Reply:
configBanjo5thString() does the needed checks to assure it is a 5 string banjo